core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -138,6 +156,7 @@ public void readFrom(StreamInput in) throws IOException {
         for (int i = 0; i < items.length; i++) {
             items[i] = Item.readItem(in);
         }
+        tookInMillis = in.readVLong();
```

Comment:
There are backwards compatibility concerns here. Consider a mixed-version cluster that contains nodes that do not know about took. You need to guard this read in a version check on the stream. For now, let's try for on or after 5.4.0. We can adjust this later if needed.